integration/spark/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableColRenameDataTypeChangeCommand.scala

```diff
@@ -169,13 +167,14 @@ private[sql] case class CarbonAlterTableColRenameDataTypeChangeCommand(
              newColumnScale)) {
           isDataTypeChange = true
         }
-        if (DataTypes.isArrayType(oldDatatype) || DataTypes.isStructType(oldDatatype)) {
+        if (DataTypes.isArrayType(oldDatatype) || DataTypes.isStructType(oldDatatype) ||
```

Comment:
This check also can be moved to some utility class and reuse in the places used in this code change

Reply:
Done